src/app/profile/edit/form/constants.ts

```diff
@@ -0,0 +1,13 @@
+import z from "zod";
+
+export const editSchema = z
+  .object({
+    name: z.string().min(4).optional().or(z.literal("")),
+    email: z.string().email().optional().or(z.literal("")),
+    password: z.string().min(8).optional().or(z.literal("")),
+    confirmPassword: z.string().min(8).optional().or(z.literal("")),
+  })
```

Comment:
Do you need the `.or(z.literal(""))`?
I will pass the validation if the user leaves the empty string in these fields.